primitives/sandbox/src/embedded_executor.rs

```diff
@@ -314,6 +315,10 @@ impl<T> super::SandboxInstance<T> for Instance<T> {
 	fn instance_globals(&self) -> Option<Self::InstanceGlobals> {
 		None
 	}
+
+	fn get_instance_ptr(&self) -> HostPointer {
+		unreachable!("Must not be called for embedded executor")
```

Comment:
To tell the truth I want to change the interface to return `Option` so it'd be a caller responsibility to process the error.

Reply:
It's not allowed to use this function for embedded. If we call it, it's very gross violation. I cannot see the case where this can help us to avoid problems. Also we now have separate extrinsic for process queue, so I think in case of errors of such level it's more suitable to stop runtime execution, fix it and then restart, than to process contract and finish execution with error.